scripts/sentry_release_sourcemaps.sh

```diff
@@ -39,4 +39,8 @@ if [ "${TRAVIS_BRANCH}" = "master" ] || [ "${TRAVIS_BRANCH}" = "devel" ]; then
       "projects":["'"${PROJECT}"'"]
   }
   '
+  sentry-cli releases -o technical-university-of-denmark -p ${PROJECT} new ${TRAVIS_COMMIT}
+  sentry-cli releases -o technical-university-of-denmark -p ${PROJECT} files \
+  ${TRAVIS_COMMIT} upload-sourcemaps --url-prefix \
```

Comment:
indent these lines please, they belong to line 43, just wrapped in multiple lines